app/ui-react/typings/rh-uxd/index.d.ts

```diff
@@ -0,0 +1,9 @@
+declare module "patternfly-react";
+declare module '@rh-uxd/integration-core/styles/*.css' {
+  const styles: any;
+  export default styles;
+}
+declare module '@rh-uxd/integration-core/styles/*.png' {
+  const images: any;
+  export default images;
+}
```

Comment:
Did you pull this package out?  Wonder do we still need these definitions and the package.json below?

Reply:
Yep, and nope! This was leftover from before I realized it could just go away completely.. I'll remove